mongodb/src/main/scala/akka/stream/alpakka/mongodb/Mongo.scala

```diff
@@ -0,0 +1,16 @@
+package akka.stream.alpakka.mongodb
+
+import com.typesafe.config.{Config, ConfigFactory}
+import org.mongodb.scala.{MongoClient, MongoDatabase}
+
+object Mongo {
+  private val config: Config = ConfigFactory.load(getClass.getClassLoader, "application.conf")
+
+  private val dbName: String = config.getString("mongo.name")
+  private val dbAddress: String = config.getString("mongo.host")
+  private val dbPort: String = config.getString("mongo.port")
+
+  val client: MongoClient = MongoClient(s"mongodb://$dbAddress:$dbPort")
```

Comment:
Should `MongoClient` just be passed as a parameter to `MongoSource.apply`, rather than doing anything with a config. Looks like `MongoClient` already provides a few different apply methods itself.